src/XamlStyler.UnitTests/TestConfigurations/Single.json

```diff
@@ -1,4 +1,5 @@
 {
   "IndentSize": 2,
-  "AttributesTolerance": 3
+  "AttributesTolerance": 3,
+  "IgnoreDesignTimeReferencePrefix": false
```

Comment:
Our documentation is not great in these test files -- for this one, we can leave the new option out.

Reply:
Corrected